### PR TITLE
Create style

### PR DESCRIPTION
**Pull request by MenendezON**
This pull request concerns the style of the card of our book.
This app is based on Redux and React

### CODE REVIEW

File: src/css/index.css
Comment (joskalenda) [inline src/css/index.css:1]: - [ ] You are is missing the font-family `Montserrat` as per the design, please make sure you add it

File: src/components/BookList.js [hunk 1]
```diff
@@ -1,34 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBookAsync, appId } from '../redux/books/booksSlice';
 
-function BookList({ book }) {
+function BookList({ book, randomNum }) {
+  const [progress, setProgress] = useState(randomNum);
```
Comment (joskalenda) [inline src/components/BookList.js:8]: - [ ] EXPECTED DESIGN



- [ ] WHAT YOU DID



1. We have some key parts that are missing in the design as the progress bar percentage, 
2. The style of the progress bar should be identical to the one above on the original template
3. The user icon on the top-right is different from the one on the original design
4. The bookstore title is not well aligned as the above is showing 



1. Make sure you fix this text brightness, I think your color is more blue than the one on the original design



1. Please add a space around your update button as per the design above

File: src/components/BookForm.js
Comment (joskalenda) [inline src/components/BookForm.js:1]: - [ ] EXPECTED DESIGN



- [ ] WHAT YOU DID


1. We have 3 inputs forms, the first one is larger than the others and is the placeholder of `Book title`

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @MenendezON , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Great readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @MenendezON  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️
- Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)✔️
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MenendezON** [comment]
Hi @ademibowale , thank you for your feedback but I'm facing a problem. This project was my core req to complete my week. I have my feedback now but it is too late to submit it completely, what can I do to not repeat this block?

**Comment by ademibowale** [comment]
@MenendezON Write the student success to assist you on that as long as your project has been approved and no activities has left uncompleted that might help you out

**Pull request merged**